server/status_test.go

```diff
@@ -219,3 +220,59 @@ func TestStatusGossipJson(t *testing.T) {
 		}
 	}
 }
+
+// TestNodeStatusResponse verifies that node status returns the expected
+// results.
+// TODO(Bram): Add more nodes.
+func TestNodeStatusResponse(t *testing.T) {
+	ts := &TestServer{}
+	ts.Ctx = NewTestContext()
+	ts.Ctx.ScanInterval = time.Duration(5 * time.Millisecond)
+	ts.StoresPerNode = 3
+	if err := ts.Start(); err != nil {
+		t.Fatal(err)
+	}
+	defer ts.Stop()
+
+	// Make sure the node is spun up.
+	ts.node.waitForScanCompletion()
```

Comment:
Twice because it ensures that a full scan of the ranges has completed as well.  I'll add a comment.